bin/middlewares/fullText_query.js

```diff
@@ -0,0 +1,80 @@
+const axios = require('axios');
+const yaml = require('node-yaml');
+
+module.exports = function (config) {
+  const geocoderUrl = config.services.geocoder.url;
```

Comment:
This `geocoder.url` is used by the client, and may not be accessibly from the backend. Similarly to "prefetch_poi", `config.server.services.idunn.url` should be preferred when it's defined.

Actually, all environments now use a Idunn route as their geocoder URL, so this `geocoder.url` configuration could be removed completely., to derive the URL from `idunn.url`.

Reply:
Ok. I'll remove the use of this here and open an issue to simplify this config later.